src/main/java/com/service/MoviesService.java

```diff
@@ -0,0 +1,30 @@
+package com.service;
+
+import com.dao.Database;
+import com.model.movies.Movies;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+
+@Service
+public class MoviesService {
```

Comment:
Would there be a benefit to introducing an interface? If so, what could it achieve?